src/pages/editor.tsx

```diff
@@ -88,6 +124,32 @@ const EditorPage = (): JSX.Element => {
             <StyledTitle>
               <h3>Step</h3>
             </StyledTitle>
+            <StyledStepSection>
+              <StyledTextInput>
+                <form>
+                  <StyledLabel>
+                    Step Title: <br />
+                  </StyledLabel>
+                  <StyledTextBox type="text" placeholder="Step Title" />
+                </form>
+              </StyledTextInput>
+              <StyledTextInput>
+                <form>
+                  <StyledLabel>
+                    Hint: <br />
+                  </StyledLabel>
+                  <StyledTextBox type="text" placeholder="Hint" />
+                </form>
+              </StyledTextInput>
```

Comment:
This might be a bit more work, but I think it would be awesome if the hint and the success message text boxes could expand when you press enter, since hints and solutions might have to be multiple lines sometimes.

Reply:
Happy to help if you run into any problems because this can get tricky sometimes.